### ISSUE TITLE
False-positive in SlevomatCodingStandard.ControlStructures.RequireMultiLineTernaryOperator

### ISSUE THREAD

**Issue by rosasurfer** [slevomat/coding-standard#1835]
Given the following sniff configuration:
```xml
<rule ref="SlevomatCodingStandard.ControlStructures.RequireMultiLineTernaryOperator">
  <properties>
    <property name="lineLengthLimit" value="100"/>
  </properties>
</rule>
```

and the following test file/code:
```php
<?php
declare(strict_types=1);

class PHPTest {

    public static function testPhpCs(): void {
        $config = isset($_GET['config']);
        ?>
        <a href="?">
           <?=$config ? 'Hide' : 'Show'?> Configuration
        </a>
        <?php
        echo '';
    }
}
````

Expected result: no error

Actual result:

<img width="960" height="260" alt="Image" src="https://github.com/user-attachments/assets/1dc4c1cf-cf7f-463e-bdf0-0d5fe491c041" />

The false-positive shows up depending on the terminating semicolon after the ternary. However, according to PHP docs for short echo syntax: a semicolon is not required before the closing PHP tag
